contracts/SlashIndicator.sol

```diff
@@ -39,6 +51,17 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     bool exist;
   }
 
+  // Proof that a validator misbehaved in fast finality
+  struct FinalityEvidence {
```

Comment:
The vote data is totally changed, doesn't it?   FinalityEvidence need change as well

Reply:
Fixed